### ISSUE TITLE
m3u8 playlist 'RESOLUTION'

### ISSUE THREAD

**Issue by ndtoan210190** [pascalbaljetmedia/laravel-ffmpeg#135]
Hi,
I'm using your lib to create m3u8 playlist for HLS. It work fine but seem missing 'RESOLUTION' and other param compare with a sample playlist.
Without 'RESOLUTION', I can't make video player with quality selector. Please help me with this, thank you.
Here is my m3u8:
#EXTM3U
```
#EXT-X-STREAM-INF:BANDWIDTH=300000
24_300.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=400000
24_400.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=500000
24_500.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1500000
24_1500.m3u8
```
and here is sample m3u8

```
#EXTM3U
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=794000,RESOLUTION=640x360,CODECS="avc1.64001f,mp4a.40.2"

#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=64000,RESOLUTION=398x224

#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=209000,RESOLUTION=416x234,CODECS="avc1.66.30,mp4a.40.2"

#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=429000,RESOLUTION=480x270,CODECS="avc1.66.30,mp4a.40.2"

#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=1196000,RESOLUTION=768x432,CODECS="avc1.64001f,mp4a.40.2"

#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=2096000,RESOLUTION=960x540,CODECS="avc1.64001f,mp4a.40.2"

#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=3096000,RESOLUTION=1280x720,CODECS="avc1.64001f,mp4a.40.2"
```

**Comment by tasarweb** [comment]
That's my problem, please help...

**Comment by hemantachhami19** [comment]
Anybody has found the way around it ?

**Comment by hemantachhami19** [comment]
@ndtoan210190 @tasarweb  Have you found some way around it. I am still stuck ?

**Comment by pascalbaljet** [comment]
I’m working on a new version of the package with support for this feature :)

**Comment by aungmyokyaw97** [comment]
Anyone?
I need to add RESOLUTION in playlist.How do I get it?

**Comment by pascalbaljet** [comment]
I've rewritten the package and it has support for this. Could you test the `v7` branch? Almost no breaking changes except for some filter stuff in HLS exports.

https://github.com/pascalbaljetmedia/laravel-ffmpeg/tree/v7

`composer require "pbmedia/laravel-ffmpeg:v7.x-dev"`

**Comment by pascalbaljet** [comment]
Fixed in `v7.0`
